core/modules/czar/Czar.cc

```diff
@@ -199,8 +200,7 @@ Czar::submitQuery(std::string const& query,
 std::string
 Czar::killQuery(std::string const& query, std::string const& clientId) {
 
-    LOGF(_log, LOG_LVL_INFO, "KILL query: '%s'" % query);
-    LOGF(_log, LOG_LVL_INFO, "client ID: '%s'" % clientId);
+    LOGS(_log, LOG_LVL_DEBUG, "KILL query: " << query << ", clientId: " << clientId);
```

Comment:
OK, we clearly need some agreement what to log at INFO level. It looks like with your changes there is nothing logged at INFO. I think I would be OK to have some per-query messages to be logged at info, plus global initialization info (not very detailed). Could we agree not to change logging levels in this ticket in revise it later in separate ticket?